core/modules/nfv_ctrl.cc

```diff
@@ -219,7 +375,10 @@ void NFVCtrl::UpdateFlowAssignment() {
   }
   bess::utils::bucket_stats.bucket_table_lock.unlock();
   
-  WriteToGurobi(total_core_count_, flow_rate_per_bucket,slo_p50_);
+  std::map<uint16_t, uint16_t> moves = longTermOptimization(flow_rate_per_bucket);
+  //Create reta table and upadate rss
+  port_->UpdateRssReta(moves);
+//  WriteToGurobi(total_core_count_, flow_rate_per_bucket,slo_p50_);
```

Comment:
Where did you call ''WriteToGurobi'?

Reply:
I was calling it previously but I have removed that code as the gurobi model needs more work.